backend/services/services.py

```
@@ -1,55 +1,94 @@
-def get_newer_posts(last_seen_post: Optional[int], db: Session):
-    if last_seen_post:
-        latest_post = db.query(Post).filter(Post.id == last_seen_post).first()
-        if latest_post:
-            return db.query(Post).filter(Post.created_at > latest_post.created_at)
-    return db.query(Post)
+# URL constants
+BASE_URL = "http://127.0.0.1:8000"
+MEDIA_URL_PREFIX = f"{BASE_URL}/uploads/media"
+DOCUMENT_URL_PREFIX = f"{BASE_URL}/uploads/document"
+
+def get_newer_posts(last_seen_post: Optional[int], db: Session) -> Any:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `get_newer_posts`

(ANN401)